src/edu/ucsb/nceas/metacat/DocumentIdQuery.java

```diff
@@ -124,7 +124,7 @@ public static void main(String[] args) {
 		String CONFIG_DIR = "lib";
 		File dirPath = new File(CONFIG_DIR);
 		try {
-			PropertyService.getInstance(dirPath.getPath() + FileUtil.getFS() + "metacat.properties");
+			PropertyService.getInstance(dirPath.getPath());
 		} catch (ServiceException ioe) {
 			System.err.println("Error in loading properties: " + ioe.getMessage());
```

Comment:
Why is ok to just fall through on exception here? Please add a comment or throw an exception.

Reply:
This is in a main() method used only for testing by hand from the command line, so it looks like the original author didn't consider a failure to be too consequential. I could add a System.exit(1) here?

Reply:
LMK what you think

Reply:
sounds fine. I ddin't realize it was in main(). Mostly it feels like these classes shouldn't have a main method, but we can just leave it be.